mobile/src/main/java/org/openhab/habdroid/background/BackgroundUtils.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2010-2018, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution, and is available
+ * at https://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.habdroid.background;
+
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
+import androidx.annotation.DrawableRes;
+import androidx.annotation.StringRes;
+import androidx.core.app.NotificationCompat;
+import androidx.core.content.ContextCompat;
+
+import org.openhab.habdroid.R;
+import org.openhab.habdroid.ui.MainActivity;
+
+import static org.openhab.habdroid.background.BackgroundTaskRetryBroadcastReceiver.OH_EXTRA_NOTIFICATION_ID;
+
+public class BackgroundUtils {
+    private static final String TAG = BackgroundUtils.class.getSimpleName();
+    public static final String WORKER_TAG_SEND_ALARM_CLOCK = "sendAlarmClock";
+    public static final String NOTIFICATION_TAG_BACKGROUND = "background";
+    public static final String NOTIFICATION_TAG_BACKGROUND_ERROR = "backgroundError";
+    public static final String CHANNEL_ID_BACKGROUND = "background";
+    public static final String CHANNEL_ID_BACKGROUND_ERROR = "backgroundError";
+    public static final int NOTIFICATION_ID_SEND_ALARM_CLOCK = 1;
+
+    /**
+     * Creates notification channels for background tasks.
+     * @param context
+     */
+    public static void createNotificationChannels(Context context) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            NotificationManager notificationManager =
+                    context.getSystemService(NotificationManager.class);
+
+            String name = context.getString(R.string.notification_channel_background);
+            String description =
+                    context.getString(R.string.notification_channel_background_description);
+            int importance = NotificationManager.IMPORTANCE_LOW;
```

Comment:
IMPORTANCE_LOW or IMPORTANCE_MIN? IOW, do we need a status bar icon here?

Reply:
I'm fine with IMPORTANCE_MIN. Actually IMPORTANCE_LOW was just for testing.

> do we need a status bar icon here?

No